server/odc-migrate/src/main/resources/migrate/common/V_4_2_3_1__alter_list_flow_instance_view.sql

```diff
@@ -0,0 +1,18 @@
+--
```

Comment:
What's this DDL used for?

Reply:
Used to list tickets about new task type, see PR description